components/Editor/services/editor-tools.ts

```diff
@@ -150,6 +153,10 @@ export const getTools = async ({
       class: TOOLS.InclineCode,
       inlineToolbar: true,
     },
+    table: {
```

Comment:
<img width="1434" alt="Screenshot 2023-07-07 at 15 04 31" src="https://github.com/Juratbek/upper_frontend/assets/55176773/38f7c4ad-23a0-47ce-a179-1c50130bf7bd">

data for table is empty

Reply:
You are viewing Preview which is the response from backend. Table data is correctly sent.